src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -426,13 +426,16 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environment and system noise. Their
-collection is RECOMMENDED, before/during/after each session. This data is stored
-inside a subject folder named `sub-emptyroom`. The `session label` SHOULD be
-that of the date of the empty-room recording (e.g. `ses-YYYYMMDD`). The
-`scans.tsv` file containing the date/time of the acquisition SHOULD also be
-included. Hence, users will be able to retrieve the empty-room recording that
-best matches a particular session with a participant, based on date/time of
+Empty-room MEG recordings capture the environment and system noise.
+Their collection is RECOMMENDED, before/during/after each session.
+This data MUST be stored inside a subject folder named `sub-emptyroom`.
+The `session label` SHOULD be that of the date of the empty-room recording
+(e.g., `ses-YYYYMMDD`).
+The task label in the `*_meg.json` file SHOULD be set to `noise`.
```

Comment:
```suggestion
The `task label` in the `*_meg.json` file SHOULD be set to `noise`.
```

Reply:
thanks @hoechenberger I committed your other changes except this one, because the code formatting would probably require a different take like `task-<label>` or similar :shrug: in this case I think it's fine to stay with "task label" as it is

Reply:
But there's also `` `session label` `` just **one line** above that one..??! 😅

Reply:
:man_facepalming: okay, didn't see that sorry